Lib/fontTools/ttLib/tables/C_F_F__2.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import print_function, division, absolute_import
+from fontTools.misc.py23 import *
+from fontTools import cffLib
+from . import DefaultTable
+
+
+class table_C_F_F__2(DefaultTable.DefaultTable):
+
+	def __init__(self, tag=None):
+		DefaultTable.DefaultTable.__init__(self, tag)
+		self.cff = cffLib.CFFFontSet()
+		self._gaveGlyphOrder = False
+
+	def decompile(self, data, otFont):
+		self.cff.decompile(BytesIO(data), otFont)
+		assert len(self.cff) < 2, "can't deal with multi-font CFF tables."
+
+	def compile(self, otFont):
+		f = BytesIO()
+		self.cff.compile(f, otFont)
+		return f.getvalue()
+
+	def haveGlyphNames(self):
+		if hasattr(self.cff[self.cff.fontNames[0]], "ROS"):
+			return False  # CID-keyed font
+		else:
+			return True
```

Comment:
> In CFF2, haveGlyphNames() should always return False.

That's also my understanding of the spec where it says "The CFF2 format does not contain glyph names".

@readroberts any reason why you left it like that?
Also it appears from `git grep "haveGlyphNames"` that this function is only defined here but not used anywhere...